src/app/plan/components/Wizard/StorageClassTable.tsx

```diff
@@ -119,7 +124,7 @@ const StorageClassTable = (props) => {
 
   if (rows !== null) {
     return (
-      <React.Fragment>
+      <>
```

Comment:
LGTM - but would like @mturley & @eriknelson opinion on switching to this syntax just so we can all agree this would be a worthwhile change. Only downside I see is if you wanted to add a key to the React.Fragment but I dont think we are doing that anywhere.

Reply:
I like the shorthand, so LGTM but I don't have a strong opinion. In the case of needing a key we can always just use React.Fragment for those specific fragments.